wiki/Manual.md

```diff
@@ -734,30 +734,27 @@ Each currency is an associative array (aka dictionary) with the following keys:
 {
     'id':      'btcusd',      // string literal for referencing within an exchange
     'symbol':  'BTC/USD',     // uppercase string literal of a pair of currencies
-    'base':    'BTC',         // uppercase string, unified base currency code, 3 or more letters
```

Comment:
We should not remove base and quote from here.

Reply:
@kroitor I hope I got it right this time